faqs/galaxy/dataset_tool_errors_metadata.md

```diff
@@ -0,0 +1,29 @@
+---
+title: How to find and correct tool errors related to Metadata?
+area: datasets   
+description:  Finding and Correcting Metadata
+layout: faq    
+contributors: [jennaj, beachyesh]
+---
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action.
+ - Incorrect autodetection of datatypes, which need manual modification
```

Comment:
There is no "." at the end of this sentence. It is recommended to be consistent.

Reply:
And I see @beatrizserrano already mentioned it! 
It is looking great and we'll be able to merge it soon! Thanks.

Reply:
Thanks!